packages/gatsby/cache-dir/ensure-resources.js

```diff
@@ -129,6 +129,8 @@ class EnsureResources extends React.Component {
 
   render() {
     if (!this.hasResources(this.state.pageResources) && isInitialRender) {
+      window.___failedResources = true
```

Comment:
there are also other places to set this flag - for example - https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L55-L60

Reply:
I don't think we should set the flag anywhere else - resources may fail from some other part, but ultimately it only matters if they fail from `EnsureResources` since this is the "final step" before actually displaying a page. Does that make sense or is there still a need to set it elsewhere?

Reply:
Right, this make sense in this context. We will reload on navigation if page resources failed to fetch